apps/docs/docs/reference/sdk/faststore-sdk.md

```diff
@@ -1,38 +1,40 @@
 # FastStore SDK
 
-:::caution WIP
-This documentation is currently under development.
-:::
+FastStore SDK provides tools that help you deal with key ecommerce features:
 
-FastStore SDK is a simple, framework-agnostic implementation of Commerce APIs to help you create your next React-based store with world-class performance in record time.
+- [Shopping cart](#shopping-cart)
+- [Session](#session)
+- [Search](#search)
+- [Analytics](#analytics)
 
-## Key features
+:::info
+Note that the SDK manages these aspects of your store in the context of shoppers' browsers. This means, for example, that functions related to cart and session, do not necessarily send requests to the [FastStore API](/reference/api/faststore-api). In this case, the goal is to manage information in the browser, that can then be validated in the platform with the appropriate [mutations](/reference/api/mutations).
+:::
 
-### Analytics
+Each of the features mentioned above has different properties and behaviors. Below you can learn more about each of them.
 
-The analytics module helps you create a simple and extensive event system to feed your data pool. It is biased towards Google Analytics 4, but supports any other analytics provider. Go to the [Analytics](/reference/sdk/analytics) page to learn more.
+## Shopping cart
 
-## Get started
+The SDK cart module controls the state of the shopping cart data structure in the shopper's browser. You can use them to add items to the cart, remove items, and clear the cart, among other tasks. Learn more about [shopping cart functions](/reference/sdk/cart/useCart).
 
-### Installation
+:::info
+Learn more about how to validate the cart in the shopper's browser against the information in the platform with [FastStore API mutations](/reference/api/mutations).
+:::
 
-Install `@faststore/sdk` as a dependency of your FastStore project via the command line:
+## Session
 
-```bash npm2yarn
-npm install @faststore/sdk
-```
+The session module manages the state of session information in the shopper's browser. This includes currency, channel, localization and shopper data. Learn more about the [session SDK module](/reference/sdk/session) data structure and usage.
```

Comment:
Capitalizar os nomes das features (Session, Search, Analytics).
Exemplo:
The Session module manages the state of session information in the shopper's browser.